.github/README.md

```diff
@@ -1,16 +1,32 @@
-# UndertaleModTool
-*(seeing such an amazing tool...<br/>
-... fills you with DETERMINATION.)*
+<p align="center">
+  <img src="images/logo.png" alt="UndertaleModTool Logo"/>
+</p>
 
-Heya. I heard you like digging deep into Undertale data so I made a tool just for you! 
+![Discord](https://img.shields.io/discord/566861759210586112?label=Discord&logo=discord&logoColor=white) ![GitHub](https://img.shields.io/github/license/krzys-h/UndertaleModTool?logo=github)
 
-**Downloads** are **HERE**: :point_right::point_right::point_right: https://github.com/krzys-h/UndertaleModTool/releases :point_left::point_left::point_left:
+<p align="center">(seeing such an amazing tool it fills you with DETERMINATION.)</p>
+<p align="center">Heya. I heard you like digging deep into Undertale/Deltarune or any GameMaker game data so I made a tool just for you!</p>
+<p align="center">
+  <img src="images/flowey.gif" alt="Flowey"/>
```

Comment:
Something the previous image showed well that this one doesn't, is that UMT is actually modifying game data.

Reply:
has i say in the server the image doesn't show anything remember that before utmt you could activate the console exclusive  content found in 1.0.8 with hex editing including the borders, if we want to add screenshots we need to show something that you can't do in Hex